db/migrate/20190725192124_change_user_organization_role_to_system_admin.rb

```diff
@@ -0,0 +1,13 @@
+class ChangeUserOrganizationRoleToSystemAdmin < ActiveRecord::Migration[5.2]
+  def up
+    rename_column :users, :organization_role, :system_admin
```

Comment:
This needs a not null constraint along with the default false.

Reply:
Updated.